src/extras/getChildByName.js

```diff
@@ -0,0 +1,29 @@
+var DisplayObject = require('../core/display/DisplayObject'),
+    Container = require('../core/display/Container')
+
+/**
+ * The instance name of the object.
+ *
+ * @member {string}
+ */
+DisplayObject.prototype.name = null;
+
+/**
+* Returns the display object in the container
+*
+* @param name {string} instance name
+* @return {DisplayObject}
+*/
+Container.prototype.getChildByName = function (name)
+{
+    for (i = 0; i < this.children.length; i++) 
```

Comment:
accidental global promotion of `i` here, jshint will tell you that when you run it (and you should :P)